specifications/passive/empatica_embraceplus_1.0.0.yml

```diff
@@ -0,0 +1,118 @@
+#====================================== EmbracePlus Wristband =====================================#
+vendor: EMPATICA
+model: Embraceplus
+version: 1.0.0
+app_provider: .empaticaembraceplus.EmpaticaEmbracePlusServiceProvider
+data:
+  - type: ACCELEROMETER
+    sample_rate:
+      frequency: 32
+    unit: ADC_COUNTS
+    processing_state: RAW
+    topic: android_empatica_embraceplus_acceleration
+    value_schema: .passive.empaticaembraceplus.EmpaticaEmbracePlusAcceleration
+  - type: BATTERY
+    sample_rate:
+      dynamic: true
+    unit: PERCENTAGE
+    processing_state: RAW
+    topic: android_empatica_embraceplus_battery_level
+    value_schema: .passive.empaticaembraceplus.EmpaticaEmbracePlusBatteryLevel
+  - type: BLOOD_VOLUME_PULSE
+    sample_rate:
+      frequency: 32
+    unit: NANO_WATT
+    processing_state: RAW
+    topic: android_empatica_embraceplus_blood_volume_pulse
+    value_schema: .passive.empaticaembraceplus.EmpaticaEmbracePlusBloodVolumePulse
+  - type: ELECTRODERMAL_ACTIVITY
+    sample_rate:
+      frequency: 4
+    unit: MICRO_SIEMENS
+    processing_state: RAW
+    topic: android_empatica_embraceplus_electrodermal_activity
+    value_schema: .passive.empaticaembraceplus.EmpaticaEmbracePlusElectroDermalActivity
+  - type: GYROSCOPE
+    sample_rate:
+      frequency: 32
+    unit: ADC_COUNTS
+    processing_state: RAW
+    topic: android_empatica_embraceplus_gyroscope
+    value_schema: .passive.empaticaembraceplus.EmpaticaEmbracePlusGyroscope
+  - type: MOVEMENT_INTENSITY  #Biomarker
+    sample_rate:
+      frequency: 32
+    unit: COUNTS
+    processing_state: RAW
+    topic: android_empatica_embraceplus_movement_intensity
+    value_schema: .passive.empaticaembraceplus.EmpaticaEmbracePlusMovementIntensity
+  - type: OXYGEN_SATURATION  #Biomarker
+    sample_rate:
+      frequency: 32
+    unit: PERCENTAGE
+    processing_state: RAW
+    topic: android_empatica_embraceplus_oxygen_saturation
+    value_schema: .passive.empaticaembraceplus.EmpaticaEmbracePlusOxygenSaturation
+  - type: PULSE_RATE  #Biomarker
+    sample_rate:
+      frequency: 32
+    unit: BEATS_PER_MIN
+    processing_state: RAW
+    topic: android_empatica_embraceplus_pulse_rate
+    value_schema: .passive.empaticaembraceplus.EmpaticaEmbracePlusPulseRate
+  - type: PULSE_RATE_VARIABILITY  #Biomarker
+    sample_rate:
+      frequency: 32
+    unit: MILLI_SEC
+    processing_state: RAW
+    topic: android_empatica_embraceplus_pulse_rate_variability
+    value_schema: .passive.empaticaembraceplus.EmpaticaEmbracePlusPulseRateVariability
+  - type: RESPIRATORY_RATE #Biomarker
+    sample_rate:
+      frequency: 32
+    unit: BREATHS_PER_MIN
+    processing_state: RAW
+    topic: android_empatica_embraceplus_respiratory_rate
+    value_schema: .passive.empaticaembraceplus.EmpaticaEmbracePlusRespiratoryRate
+  - type: SLEEP_DETECTION  #Biomarker
+    sample_rate:
+      frequency: 32
```

Comment:
Again not sure sleep biomarkers would be delivered with such high frequency. I would just change this to dynamic.